package.json

```diff
@@ -1,7 +1,13 @@
 {
   "name": "nuxt-todo",
   "version": "1.0.0",
+  "description": "A simple todo app with Nuxt.js",
   "private": true,
+  "engines": {
+    "node": "16.19.0",
+    "npm": "8.19.3",
+    "yarn": "1.22.19"
+  },
```

Comment:
Although this *IS* this best practice, to set up specific versions, we also use a more relaxed version like `>=16 <17` or `=16.x` so that we can do minor updates of node without changing this.

Reply:
resolved